src/app.js

```diff
@@ -10,12 +10,13 @@ const loadConfig = require('./lib/config')
 const RoutingTables = require('./lib/routing-tables')
 const RouteBuilder = require('./lib/route-builder')
 const RouteBroadcaster = require('./lib/route-broadcaster')
+const CurveCache = require('./lib/curve-cache')
```

Comment:
`Route.curve` is the current curve. The `CurveCache` records liquidity-quotes until they have expired, so that they can be honored during payment.